core/src/main/scala/org/apache/spark/deploy/history/HistoryPage.scala

```diff
@@ -168,6 +174,10 @@ private[history] class HistoryPage(parent: HistoryServer) extends WebUIPage("")
         "-"
       }
     val lastUpdated = UIUtils.formatDate(attempt.lastUpdated)
+    var someAttemptCompleted = false
+    info.attempts.foreach{ attempt =>
```

Comment:
I think this can be `val someAttemptCompleted = info.attempts.exists(_.completed)`